lib/routes/ecnu/jwc.ts

```diff
@@ -0,0 +1,74 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const type = (filename) => filename.split('.').pop();
+
+export const route: Route = {
+    path: '/jwc',
+    categories: ['university'],
+    example: '/ecnu/jwc',
+    parameters: {},
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: {
+        source: ['www.jwc.ecnu.edu.cn', 'www.ecnu.edu.cn'],
+        target: '/tzgg',
+    },
+    name: '教务处通知',
+    maintainers: ['markbang'],
+    handler: async () => {
+        const baseUrl = 'http://www.jwc.ecnu.edu.cn/';
+
+        const response = await got(`${baseUrl}tzggwwxsgg/list.htm`);
+        const $ = load(response.data);
+        const links = $('.col_news_con ul > li')
```

Comment:
```suggestion
        const links = $('.col_news_con ul.news_list > li')
```
otherwise it will select `.pages_count`, `page_nav` and `page_jump` as well.